docs/recipes/install/rocky9/x86_64/confluent/slurm/steps.tex

```diff
@@ -0,0 +1,309 @@
+\newcommand{\clean}{dnf clean expire-cache}
+\newcommand{\chrootclean}{dnf --installroot=\$CHROOT clean expire-cache}
+\newcommand{\install}{dnf -y install}
+\newcommand{\chrootinstall}{nodeshell compute dnf -y  install}
+\newcommand{\groupinstall}{dnf -y groupinstall}
+\newcommand{\groupchrootinstall}{nodeshell compute dnf -y groupinstall}
+\newcommand{\remove}{dnf -y remove}
+\newcommand{\upgrade}{dnf -y upgrade}
+\newcommand{\chrootupgrade}{dnf -y --installroot=\$CHROOT upgrade}
+\newcommand{\tftppkg}{syslinux-tftpboot}
+\newcommand{\beegfsrepo}{https://www.beegfs.io/release/beegfs\_7.2.1/dists/beegfs-rhel8.repo}
+
+% boolean for os-specific formatting
+\toggletrue{isCentOS}
+\toggletrue{isCentOS_ww_slurm_x86}
+\toggletrue{isSLURM}
+\toggletrue{isx86}
+\toggletrue{isCentOS_x86}
+
+\begin{document}
+\graphicspath{{common/figures/}}
+\thispagestyle{empty}
+
+% Title Page
+\input{common/title}
+% Disclaimer
+\input{common/legal}
+
+\newpage
+\tableofcontents
+\newpage
+
+% Introduction  --------------------------------------------------
+
+\section{Introduction} \label{sec:introduction}
+\input{common/install_header}
+\input{common/intro} \\
+
+\input{common/base_edition/edition}
+\input{common/audience}
+\input{common/requirements}
+\input{common/inputs}
+
+
+
+% Base Operating System --------------------------------------------
+
+\section{Install Base Operating System (BOS)}
+\input{common/bos}
+
+%\clearpage
+% begin_ohpc_run
+% ohpc_validation_newline
+% ohpc_validation_comment Disable firewall
+\begin{lstlisting}[language=bash,keywords={}]
+[sms](*\#*) systemctl disable firewalld
+[sms](*\#*) systemctl stop firewalld
+\end{lstlisting}
+% end_ohpc_run
+
+% ------------------------------------------------------------------
+
+\section{Install \Confluent{} and Provision Nodes with BOS} \label{sec:provision_compute_bos}
+\input{common/confluent_compute_bos_intro}
+
+\subsection{Enable \Confluent{} repository for local use} \label{sec:enable_confluent}
+\input{common/enable_confluent_repo}
+
+\subsection{Add provisioning services on {\em master} node} \label{sec:add_provisioning}
+\input{common/install_provisioning_confluent_intro}
+
+\vspace*{-0.15cm}
+\subsection{Complete basic \Confluent{} setup for {\em master} node} \label{sec:setup_confluent}
+\input{common/confluent_setup}
+
+\subsection{Define {\em compute} image for provisioning}
+\input{common/confluent_init_os_images_rocky}
+
+\vspace*{0.9cm}
+\subsection{Add compute nodes into \Confluent{} database} \label{sec:confluent_add_nodes}
+\input{common/add_confluent_hosts_intro}
+
+%\vspace*{-0.25cm}
+\subsection{Boot compute nodes} \label{sec:boot_computes}
+\input{common/reset_computes_confluent}
+
+\section{Install \OHPC{} Components} \label{sec:basic_install}
+\input{common/install_ohpc_components_intro}
+
+\subsection{Enable \OHPC{} repository for local use} \label{sec:enable_repo}
+\input{common/enable_local_ohpc_repo_confluent}
```

Comment:
I am not aware of the history behind this line from the xcat recipe.In all other recipes we enable the OpenHPC repository by installing the OpenHPC release RPM which enabled a dnf repository from the OpenHPC repository server. Hardcoding the downloading of the repository tar files feels unnecessary especially as we do not do it at all in any of our current testing. Please try to work with the online repository if that would work for you.

If you need it for your testing we should put it behind some variable, so that it can be disabled.

Is this strictly necessary for you or can you work with the online repository?

Reply:
Sure noted, l will look to work with the online repo.